src/EditorFeatures/CSharpTest/RawStringLiteral/RawStringLiteralCommandHandlerTests.cs

```diff
@@ -296,6 +312,32 @@ public void TestGenerateWithInterpolatedString()
 @"var v = $""""""$$""""""");
         }
 
+        [WpfFact, WorkItem(66538, "https://github.com/dotnet/roslyn/issues/66538")]
+        public void TestGenerateWithInterpolatedString_TwoDollarSigns()
+        {
+            using var testState = RawStringLiteralTestState.CreateTestState(
+"""var v = $$""[||]""", withSpansOnly: true);
+
+            testState.SendTypeChar('"');
+            testState.AssertCodeIs(
+""""
+var v = $$"""[||]"""
+"""", withSpansOnly: true);
+        }
```

Comment:
can you add a test where one more quote is typed ad this point?

Reply:
Sure. Added!